packages/block-editor/src/components/block-toolbar/utils.js

```diff
@@ -17,25 +18,27 @@ const {
 const DEBOUNCE_TIMEOUT = 250;
 
 /**
- * Hook that creates a showMover state, as well as debounced show/hide callbacks.
+ * Hook that creates debounced activate/deactivate callbacks.
  *
  * @param {Object}   props                       Component props.
  * @param {Object}   props.ref                   Element reference.
- * @param {boolean}  props.isFocused             Whether the component has current focus.
- * @param {number}   [props.debounceTimeout=250] Debounce timeout in milliseconds.
+ * @param {boolean}  props.isFocused             Whether the component has
+ *                                               current focus.
+ * @param {number}   [props.debounceTimeout=250] Debounce timeout in
+ *                                               milliseconds.
  * @param {Function} [props.onChange=noop]       Callback function.
  */
-export function useDebouncedShowMovers( {
+export function useDebouncedActivateCallbacks( {
```

Comment:
This is now only being used for the block outline shown when the block icon is hovered. I could follow up with another PR to remove the debouncing entirely, if desired.